lib/react/rails/controller_renderer.rb

```diff
@@ -0,0 +1,18 @@
+class React::ControllerRenderer
```

Comment:
Can this be named `React::Rails::ControllerRenderer`? That would match the file path (and seems like the right name for it)